File: api/code-samples/python/eth/block/getBlockByHash.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blochHash = "0xb8deae63002d2b6aa33247c8ef545383ee0fd2282ac9b49dbbb74114389ddb5c"
transactionObject = True

sdk = OpenSDK(host)
eth_response = sdk.eth.get_block_by_hash(blochHash, transactionObject)

print(json.loads(eth_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getBlockByHash.js

const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));


    const blockHash = '0xba647d41423faeebe8a7c64737d284fc2eba6f0388a3e1ebf6243db509ec1577'
    const returnTransactionObject = true
    
    sdk.klay.getBlockByHash(blockHash, returnTransactionObject, {}, (err, data, response) => {
        console.log(data);
    });
}
)()